maxim/logger/pydantic_ai/client.py

```
@@ -0,0 +1,737 @@
+)
+
+_last_llm_usages = {}
+_agent_span_ids = {}
+_session_trace = None  # Global session trace
+
+_global_maxim_trace: contextvars.ContextVar[Union[Trace, None]] = (
+    contextvars.ContextVar("maxim_trace_context_var", default=None)
+)
+
+
+def get_log_level(debug: bool) -> int:
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)